easybuild/easyconfigs/b/Bazel/Bazel-2.0.0-CrayGNU-19.10.eb

```diff
@@ -0,0 +1,27 @@
+# @author: sarafael, gppezzi, marcelsc
+
+easyblock = "Binary"
+
+name = 'Bazel'
+version = '2.0.0'
+
+homepage = 'http://baze.io/'
+description = """Correct, reproducible, fast builds for everyone"""
+
+toolchain = {'name': 'CrayGNU', 'version': '19.10'}
+
+osdependencies = ['java']
+
+sources = ['bazel-%(version)s-installer-linux-x86_64.sh']
+source_urls = ['https://github.com/bazelbuild/bazel/releases/download/%(version)s']
+
+install_cmd = 'chmod +x bazel-%(version)s-installer-linux-x86_64.sh && ./bazel-%(version)s-installer-linux-x86_64.sh --bin=%(installdir)s --base=%(installdir)s'
+
+local_files_to_copy = [(['output/bazel'], 'bin')]
```

Comment:
Since it's a local variable it has no effect, right? Maybe we should delete this completely